youtube_dl/extractor/aznude.py

```diff
@@ -0,0 +1,119 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+try:
+    from urlparse import urlparse
+except ImportError:
+    from urllib.parse import urlparse
+
+import re
+
+
+class AZNudeIE(InfoExtractor):
+    IE_NAME = 'aznude'
+    _VALID_URL = r'https?://(?:www\.)?aznude\.com/(?:mrskin|azncdn)/[^/?]+/[^/?]+/(?P<id>.*)\.html'
+    _TEST = {
+        'url': 'https://www.aznude.com/mrskin/marisatomei/loiteringwithintent/loiteringwithintent-mcnallytomei-hd-01-hd.html',
+        'md5': '28973bf7b818edfe55677b67bc073e40',
+        'info_dict': {
+            'id': 'loiteringwithintent-mcnallytomei-hd-01-hd',
+            'ext': 'mp4',
+            'title': 'Marisa Tomei in Loitering With Intent - 01',
+            'thumbnail': 'https://cdn1.aznude.com/marisatomei/loiteringwithintent/LoiteringWithIntent-McNallyTomei-HD-01-gigantic-4.jpg',
+            'description': 'Watch Marisa Tomei\'s Breasts scene on AZNude for free (22 seconds).',
+        },
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        numeric_id = '-'.join(re.findall(r'(?P<num>(?:s\d+e\d+)|(?:\d+[xX]\d+)|(?:\d+))', video_id))
+        webpage = self._download_webpage(url, video_id)
+
+        jwplayer_data = self._find_jwplayer_data(webpage)
+        parsed_formats = self._parse_jwplayer_data(jwplayer_data, video_id, require_title=False)['formats']
+
+        for format in parsed_formats:
+            url = format['url']
+
+            if url.endswith('-lo.' + format['ext']):
+                format['format'] = 'Low Quality'
+                format['format_id'] = 'LQ'
+                format['quality'] = 1
+                format['resolution'] = '640x360'
+                format['format_note'] = '360p video with 64 kbps audio'
+
+            elif url.endswith('-hi.' + format['ext']):
+                format['format'] = 'High Quality'
+                format['format_id'] = 'HQ'
+                format['quality'] = 2
+                format['resolution'] = '640x360'
+                format['format_note'] = '360p video with 128 kbps audio'
+
+            elif url.endswith('-hd.' + format['ext']):
+                format['format'] = 'High Definition'
+                format['format_id'] = 'HD'
+                format['quality'] = 2
+                format['resolution'] = '1280x720'
+                format['format_note'] = '720p video with 128 kbps audio'
```

Comment:
Apparently, the audio bitrate is not consistent for any quality level, probably because they're using VBR. I'll remove the audio bitrate from the description entirely.